lib/buffer.js

```diff
@@ -94,17 +99,19 @@ function alignPool() {
   }
 }
 
+const bufferDeprecationWarning =
+    deprecate(() => {}, 'Buffer constructor is deprecated. ' +
+      'Use Buffer.from, Buffer.allocUnsafe or Buffer.alloc instead.');
```

Comment:
I would prefer the order here to be `Buffer.from, Buffer.alloc or Buffer.allocUnsafe` instead, because the latter should be used when people are better aware of what they are doing (i.e. are sure that they don't leak the resulted Buffer in an (partially) unitialized state in any code branch).